files/optics.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Back-end Analysis}
+This chapter gives insight into the performance of the back-end application.
+
+\section{Optics}
+Optics is an analysis tool build by the Meteor developer team for use on
+GraphQL servers. They currently support Express, Hapi, Koa and Ruby servers. It is a
+commercial product but also contains a free plan for developers.
+
+Optics was added to the project to get insights into the performance of the
+different queries the user can perform. This data can be used to pinpoint
+performance bottlenecks and in general pinpoint what the different fields
+returned costs performance wise.
+
+\subsection{Performance analysis}
+Below is a screenshot from Optics that shows the analysis for querying after
+the viewer, including the user but excluding the users measurements.
```

Comment:
Maybe wrap "viewer" in verb: `\verb+viewer+` to make it clear it's an entity type. Same with the other references below.